docs/introduction.md

```diff
@@ -1,13 +1,29 @@
 ---
-id: introduction 
-title: Quick Start
-sidebar_label: Quick Start
+id: introduction
+title: Introduction to Avolites Titan
+sidebar_label: Introduction to Avolites Titan
 ---
 
-This section is a quick guide on how to do most things on Avolites Titan
-consoles. Each section has a link to take you to the appropriate manual
-section for more details.
+Welcome to Avolites Titan! This manual aims to you tell you everything
+there is to know about operating your Avolites console (that's quite a lot).
 
-Softkeys (the A-G menu buttons next to the screen) are shown like 
-\[This\]. Physical labelled buttons on the console are
-shown like \<This\>.
+If you're already familiar with using lighting consoles but new to Titan, 
+the Quick Start section tells you how to do the most common things.
+
+After that we try to take you through all the things you will need to know in the order
+that you're most likely to need them, so we start with how to connect up the console,
+then look at patching lights. After that we go through how to control the different
+attributes of the lights, then how to store frequently-used settings in palettes.
+Titan has some great built-in effects, so we look at those, then how to build a show
+by recording cues, chases and cue lists. Later there are sections on Capture Visualiser, 
+the video integration system Synergy, help when running a show, remote console control,
+and system options. 
```

Comment:
It would be good if these linked to the pages in the manual

Reply:
Links added